tests/verify.test.js

```diff
@@ -0,0 +1,90 @@
+import chai from 'chai';
+import chaihttp from 'chai-http';
+import createToken from '../src/helpers/tokens/generate.token';
+import env from '../src/configs/environments';
+import server from '../src/index';
+import status from '../src/helpers/constants/status.codes';
+import errMessages from '../src/helpers/constants/error.messages';
```

Comment:
@truestbyheart remember - meaningful variable names always.  Why not use `errorMessages` here?